rest/resource-server/src/docs/asciidoc/releases.adoc

```diff
@@ -49,6 +49,24 @@ include::{snippets}/should_document_get_releases_with_fields/http-response.adoc[
 include::{snippets}/should_document_get_releases_with_fields/links.adoc[]
 
 
+[[resources-releases-list-by-name]]
+==== Listing releases by name
+
+A `GET` request will list all of the service's releases by name.
```

Comment:
It will improve the readability if we can add the parameter name in description as well.

Reply:
Changes made as suggested.